section4/exercises/tweet.js

```diff
@@ -7,7 +7,34 @@ In the space below, add to the existing skeleton of a Tweet class.
 Create several instances of your Tweet and log them to the console. Make sure the
 tweet object instances _behave_ as expected.
 */
-
+//class Tweet. Object instances: annoucement, quote, warning
 class Tweet {
+  constructor(author,content,timeStamp,numberOfLikes,comments){
+    this.author = author;
+    this.content = content;
+    this.timeStamp = timeStamp
+    this.numberOfLikes = numberOfLikes
+    this.comments = comments
+  }
+  addLikes(){
+    this.numberOfLikes = (numberOfLikes++);
+  }
+    addComments(){
+      this.comments = (comments++);
+    }
+
+}
+var annoucement = new Tweet("National Park Services", "Park is closed today due to weather", "10:55AM 10/20/2019", 20, ["Bummer!", "We'll have to come back.", "Next time!"] )
+
+var quote = new Tweet("Inspirational Quotes", "Remember, Hope is a good thing, maybe the best of things, and no good thing ever dies.", "9:30PM 9/13/2005", 40, ["One of my favorite quotes!", "Love this book!", "Rita Hayworth & Shawshank Redemption.."])
+
+var warning = new Tweet("Weather Channel", "Winter weather storm advisory", "6:30AM 10/24/2020", 50, ["Time to hunker down", "Get the shovels ready!", "Thanks for the heads up!"])
+
+
+console.log(annoucement);
```

Comment:
Did you happen to log what happens if you invoke the methods `addLikes` or `addComments` here?  I think you'll find those will both trigger errors due to `numberOfLikes` and `comments` being undefined.

Reply:
@mschae16 
You're right, they did trigger an error. I went back and figured out what I was missing and needed to adjust. My updated methods are below, and I've also attached a screenshot of the changes I made in the code. Please let me know if you need anything else. 
![Screen Shot 2020-11-11 at 3 07 57 PM](https://user-images.githubusercontent.com/71860165/98870931-4a17c000-2431-11eb-8af5-27258923fb9b.png)

Reply:
Great updates here!  Another way to write `this.numberOfLikes = this.numberOfLikes + 1` is with the incrementor operator, like so: `this.numberOfLikes++`.  No need to update that here though, just something to keep in mind for future.